thunder/benchmarks/benchmark_litgpt.py

```diff
@@ -393,7 +401,15 @@ def setup_compile(self, model):
 
                 executors.insert(0, transformer_engine_ex)
 
-            model = thunder.jit(model, executors=executors)
+            transforms = []
+            if self.distributed_mode == "fsdp":
+                from thunder.transforms import MaterializationTransform
```

Comment:
I probably need to learn what this is doing but if there could be a ELI5 on the PR for this change, that would be great.

Reply:
Without this transform, the materialization of parameters is deferred to the first forward call, after the optimizer's construction while the fused optimizer requires all the params to be materialized on a certain device. This `MaterializationTransform` is helpful in this regard. Of course, I'll add some comment to the code but just so you know.